lib/child_process.js

```diff
@@ -545,8 +545,7 @@ function copyProcessEnvToEnv(env, name, optionEnv) {
 }
 
 function normalizeSpawnArguments(file, args, options) {
-  validateString(file, 'file');
-  validateArgumentNullCheck(file, 'file');
+  file = getValidatedPath(file, 'file');
```

Comment:
`validatePath` inside `getValidatedPath` also accepts buffers. Can you add a test for it as well?

Reply:
Nice catch! I guess it should be restricted to string or stringified URL, at least for now.